os/kernel/mqueue/mq_sndinternal.c

```diff
@@ -214,9 +217,11 @@ FAR struct mqueue_msg_s *mq_msgalloc(void)
 			mqmsg = (FAR struct mqueue_msg_s *)kmm_malloc((sizeof(struct mqueue_msg_s)));
 
 			/* Check if we got an allocated message */
-
-			ASSERT(mqmsg);
```

Comment:
Could you check line 175? Description of this function says assert on failure. Could you modify that according to our new policy?

Reply:
Modified. Thanks.